github_tui.opam.template

```diff
@@ -1,3 +1,4 @@
 pin-depends: [
   ["shape-the-term.dev" "git+https://github.com/jmcavanillas/shape-the-term"]
+  ["ansifmt.dev" "git+https://github.com/qexat/ansifmt"]
```

Comment:
Depending on a GitHub version until it's merged to opam-repository:

- https://github.com/ocaml/opam-repository/pull/27357

Reply:
v0.2.0 was just released, so I updated that